src/control/server/harness.go

```diff
@@ -36,16 +37,20 @@ import (
 // IOServerHarness is responsible for managing IOServer instances
 type IOServerHarness struct {
 	sync.RWMutex
-	log       logging.Logger
-	instances []*IOServerInstance
-	started   bool
+	log            logging.Logger
+	instances      []*IOServerInstance
+	started        bool
+	startInstances chan struct{}
+	ready          chan struct{}
 }
 
 // NewHarness returns an initialized *IOServerHarness
 func NewIOServerHarness(log logging.Logger) *IOServerHarness {
 	return &IOServerHarness{
-		log:       log,
-		instances: make([]*IOServerInstance, 0, 2),
+		log:            log,
+		instances:      make([]*IOServerInstance, 0, 2),
```

Comment:
No need to make a change in this PR, but just wondering - is there a const defined somewhere for max number of instances?

Reply:
I doubt it given @mjmac hardcoded it here but I'm sure we can create one

Reply:
There is a constant defined at https://github.com/daos-stack/daos/blob/master/src/control/server/server.go#L59 ... Wouldn't hurt to use that here. Remember though that this value is just a starting point rather than a hard cap on the slice length.